src/ex_run.rs

```diff
@@ -20,6 +20,26 @@ pub fn delete_all_results(ex_name: &str) -> Result<()> {
     Ok(())
 }
 
+pub fn delete_result(ex_name: &str, tc: Option<Toolchain>, crate_: &ExCrate) -> Result<()> {
+    let config = &load_config(ex_name)?;
+
+    let tcs: [Toolchain; 1];
```

Comment:
I think so. Otherwise, the reference to the array in the single-toolchain case doesn't live long enough. This could probably be improved with something like https://crates.io/crates/ref_slice but I'll do that in a follow up.